src/test/regress/expected/insert_select_repartition.out

```diff
@@ -1216,6 +1216,7 @@ ON CONFLICT(c1, c2, c3, c4, c5, c6)
 DO UPDATE SET
  cardinality = enriched.cardinality + excluded.cardinality,
  sum = enriched.sum + excluded.sum;
+DEBUG:  rehashing catalog cache id 14 for pg_opclass; 17 tups, 8 buckets at character 224
```

Comment:
If we are ignoring this, then this shouldn't be here I guess?

Reply:
If we remove the dollar sign in the regular expression, it will start to work I think.

`/^DEBUG:  rehashing catalog cache id [0-9]+/d`

Reply:
Does that matter whether it is in the output or not given that we ignore it?

Reply:
Currently we don't ignore it I think, because the current regex doesn't match anything

Reply:
So if for a run:
```
  rehashing catalog cache id 14 for pg_opclass; 17 tups, 8 buckets at character 224
```
changes, it will start failing I guess.